packages/data-service-generator/src/server/resource/create-resource.ts

```diff
@@ -63,6 +64,19 @@ async function createResourceModules(entity: Entity): Promise<ModuleMap> {
 
   const [controllerModule, controllerBaseModule] = controllerModules.modules();
 
+  const controllerGrpcModules =
+    (appInfo.generateGrpc &&
```

Comment:
This seems to be the only touch-point between the old dsg and this new code. 
Is there any blocker to move all the modules generation (with templates and co.) in the plugin?

Reply:
We decided to remove this fromt he appInfo and calculate it directly into the dsg-context in prepare-context.